CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -171,9 +171,13 @@ CF_PRIVATE CFIndex __CFActiveProcessorCount();
 #define __builtin_unreachable() do { } while (0)
 #endif
 
-#if defined(__i386__) || defined(__x86_64__)
-    #if defined(__GNUC__)
-        #define HALT do {asm __volatile__("int3"); kill(getpid(), 9); __builtin_unreachable(); } while (0)
+#if defined(__i386__) || defined(__x86_64__) || defined(__s390x__)
```

Comment:
We have a few architecture checks across CF but we don't usually check for s390x there. Is neither i386 nor x86_64 defined on that platform?